packages/data/src/registry.js

```diff
@@ -95,18 +101,20 @@ export function createRegistry( storeConfigs = {} ) {
 	/**
 	 * Registers a new sub-reducer to the global state and returns a Redux-like store object.
 	 *
-	 * @param {string} reducerKey Reducer key.
-	 * @param {Object} reducer    Reducer function.
+	 * @param {string}  reducerKey  Reducer key.
+	 * @param {Object}  reducer     Reducer function.
+	 * @param {boolean} persist     Should the reducer be persisted.
 	 *
 	 * @return {Object} Store Object.
 	 */
-	function registerReducer( reducerKey, reducer ) {
+	function registerReducer( reducerKey, reducer, persist = false ) {
```

Comment:
Not really loving how this API is evolving. At a glance, I have no idea what `false` is in the following line of code:

```js
wp.data.registerReducer( 'core/my-store', reducer, false );
```

If we're going to add additional options, maybe we should just have this be part of an options object. At least that way, it'll be clearer what we're doing with named keys, and allow room to grow, where order of the arguments is already not important.

Reply:
I don't like it a lot as well but I consider these `register*` functions as internal functions that should be deprecated at some point in favor of `registerStore` (especially registerReducer) and the `registerStore` function already has the options object.

Reply:
Yes, I had also some concerns regarding this flag, but we solved a bunch of other issues. I agree that an object would be easier to read.